python/lsst/daf/butler/registry/datasets/byDimensions/_manager.py

```diff
@@ -273,11 +324,38 @@ def getCollectionSummary(self, collection: CollectionRecord) -> CollectionSummar
         # Docstring inherited from DatasetRecordStorageManager.
         return self._summaries.get(collection)
 
-    @classmethod
-    def currentVersion(cls) -> Optional[VersionTuple]:
-        # Docstring inherited from VersionedExtension.
-        return _VERSION
-
     def schemaDigest(self) -> Optional[str]:
         # Docstring inherited from VersionedExtension.
         return self._defaultSchemaDigest(self._static, self._db.dialect)
+
+    _version: VersionTuple
+    """Schema version for this class."""
+
+    _storageClass: Type[ByDimensionsDatasetRecordStorage]
```

Comment:
I'm a bit confused by the term "storageClass" being used here to store something that isn't a `StorageClass`.

Reply:
storageClass in this context means class (type) which implements `ByDimensionsDatasetRecordStorage` interface, but I see that it can be confused with butler StorageClass. I'll try to find a more distinct name for it.

Reply:
I renamed it to `_recordStorageType`.